src/topics/group-generator/group-generator.ts

```diff
@@ -176,9 +178,13 @@ export class GroupGeneratorService {
       );
     }
 
+    const endGeneration = Date.now();
+    const executionTime = (endGeneration - startGeneration) / 1000;
```

Comment:
Missing Math.floor?

Reply:
Why putting a Math.floor? We don't want to have the decimals of the time?

Reply:
only in second (we don't need more precision than that)